lib/ts/version.ts

```diff
@@ -12,7 +12,7 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-export const version = "16.6.5";
+export const version = "16.6.6";
```

Comment:
new version should be 16.6.7 since we are in the process of releasing 16.6.6 already.